src/flow/index.js

```diff
@@ -1,47 +1,24 @@
-import React, { createElement } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-
 import { ReactFlowProvider } from 'react-flow-renderer'
-import AutoSizer from 'react-virtualized-auto-sizer'
-import { styled, setup } from 'goober'
-
 import DAGNode from './dag-node'
 import Layout from './layout'
-import Panel from './panel'
-
-
-setup(createElement)
+import { ThemeProvider } from '@material-ui/core/styles'
+import Theme from '@eqworks/react-labs/dist/theme'
+import { useWindowDimensions } from './hooks'
 
 export const nodeTypes = { DAGNode }
 
-export const Container = styled('div')`
-  width: inherit;
-  height: inherit;
-  display: flex;
-  font-family: "OpenSans", Helvetica, sans-serif;
-`
-
-export const FlowContainer = styled('div')`
-  flex: 8;
-  height: inherit;
-  margin: auto;
-  padding-left: 1rem;
-`
-
-const Flow = ({ data, config }) => (
-  <Container>
-    <ReactFlowProvider>
-      <Panel />
-      <FlowContainer>
-        <AutoSizer>
-          {({ width, height }) => (
-            <Layout data={data} config={config} width={width} height={height} />
-          )}
-        </AutoSizer>
-      </FlowContainer>
-    </ReactFlowProvider>
-  </Container>
-)
+const Flow = ({ data, config }) => {
+  const { width, height } = useWindowDimensions()
+  return (
+    <ThemeProvider theme={Theme}>
```

Comment:
this would create some performance overhead. rather apply the theme in snoke

Reply:
Excluding `<ThemeProvider>` Storybook won't be able to render `Flow` as `Theme` props are used inside of `makeStyles()`. Plus imported `<Flow>` needs to be wrapped with `<ThemeProvider>`

Reply:
Not a while ago, Sam changed the theme application mechanism to the one requires ThemeProvider on the very top of the page
https://eqworks.slack.com/archives/GSNL88FQW/p1599146711063200

Reply:
yea that's what i meant, this is anyway provided in snoke, like https://github.com/EQWorks/snoke/blob/6598ab3bef15933e50da75af310e7a93f05487e9/src/containers/new-marketplace/index.js#L45 (where in this case we can apply at the builder view root). that would be more natural as "theme" goes.

Reply:
Then I'll need to wait for Sam to merge his theme export PR, then move ThemeProvider up to preview.js

Reply:
you don't need to wait. the react-labs `ThemeProvider` is the same thing as the MUI one, with its `theme` prop defaults to our theme.

you can do that by doing:

```jsx
import { ThemeProvider } from '@material-ui/core/styles'

import DefaultTheme from '@react-labs/dist/theme'

// in preview
<ThemeProvider theme={DefaultTheme}>
  ...
</ThemeProvider>
```